solidity/dashboard/src/hooks/useEtherscanUrl.js

```diff
@@ -1,10 +1,8 @@
-import { useContext } from "react"
-import { Web3Context } from "../components/WithWeb3Context"
+import { getFirstNetworkIdFromArtifact } from "../contracts"
 
+const networkID = getFirstNetworkIdFromArtifact()
```

Comment:
Would it be better ot use `getChainId` from `utils`? Feels like we're using it in similar scenarios elsewhere, and it lets us wrap the `getFirstNetworkIdFromArtifact` behavior centrally in case we decide to change it later.